nova/src/test.rs

```diff
@@ -14,3 +14,127 @@ impl<C: CircuitDriver> Function<C> for ExampleFunction<C> {
         DenseVectors::new(vec![next_z])
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::hash::{MimcRO, MIMC_ROUNDS};
+    use crate::prover::tests::example_prover;
+    use crate::relaxed_r1cs::{RelaxedR1csInstance, RelaxedR1csWitness};
+    use crate::{RelaxedR1cs, Verifier};
+    use bn_254::{Fq, Fr};
+    use zkstd::circuit::CircuitDriver;
+    use zkstd::matrix::DenseVectors;
+    use zkstd::r1cs::test::example_r1cs;
+
+    /// Verify committed folded instance relations.
+    /// Does not open the commitments, but just checks that
+    /// the given relaxed instances (instance1, instance2)
+    /// when folded result in the folded committed instance values.
+    pub fn verify_folded_instance<C: CircuitDriver>(
```

Comment:
do we use from other crate?
pub is not necessary I think

Reply:
If this function is not used from other place, we don't have to implement this as function